kumo-cli/src/main/java/com/kennycason/kumo/cli/KumoCli.java

```diff
@@ -110,7 +110,14 @@ private void buildLayeredWordCloud() {
             wordCloud.setColorPalette(i, new ColorPalette(cliParameters.getLayeredColors().get(i)));
             wordCloud.setFontScalar(i, buildFontScalar(cliParameters.getFontScalarType()));
             wordCloud.setPadding(i, cliParameters.getPadding());
+
+            // Normal Rendering
             wordCloud.setKumoFont(i, buildKumoFont(cliParameters.getFontWeight()));
+            // Render Chinese Characters on Windows
+//            wordCloud.setKumoFont(i, new KumoFont(new File("C:\\Windows\\Fonts\\msyh.ttc")));
```

Comment:
if you were to uncomment these lines, does:
```wordCloud.setKumoFont(i, new KumoFont(new File("C:\\Windows\\Fonts\\msyh.ttc")));```
correctly import the font file as expected?